mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -88,3 +146,19 @@ def get_endpoint(
         end=end,
         features=features,
     )
+
+
+def get_or_raise(dictionary: dict, key: str):
+    if key not in dictionary:
+        raise MLRunInvalidArgumentError(
+            f"Required argument '{key}' missing from json payload: {dictionary}"
+        )
+    return dictionary[key]
+
+
+def verify_endpoint(project, endpoint_id):
+    endpoint_id_project, _ = endpoint_id.split(".")
+    if endpoint_id_project != project:
+        raise MLRunConflictError(
+            f"project: {project} and endpoint_id: {endpoint_id} mismatch."
+        )
```

Comment:
Well it is a part of the ID by design, but this is making this validating code depended on the implementation of the `id` meaning, it will break if the implementation changes. This is actually remaining code from before we passed the access key in the headers of the request. This is actually now obsolete.